CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Add `mapping_tool_id` slot to the `Mapping` and `MappingSet` classes ([issue](https://github.com/mapping-commons/sssom/issues/449)).
 - Add `record_id` slot to the `Mapping` class ([issue](https://github.com/mapping-commons/sssom/issues/359)).
 - Change all URI-typed slots to clarify that they expect _non-relative_ URIs as values ([issue](https://github.com/mapping-commons/sssom/issues/448)).
+- Add `curation_rule` and `curation_rule_text` to the `MappingSet` class and made propagatable ([issue](https://github.com/mapping-commons/sssom/issues/464)).
```

Comment:
can you give an example that demonstrates what a curation rule text is, and it being used in a propgatable way?

Reply:
I think you misunderstand. You're requesting to modify the spec in this PR without providing an example of how that change might look in practice

Even if we throw out the idea that the example should be meaningful, I insist that you at least put _something_ that shows `curation_rule_text` being used in the metadata at the top of a SSSOM document